specs/binding-http.spec/src/main/resources/META-INF/zilla/http.idl

```diff
@@ -49,4 +49,77 @@ scope http
             HttpHeader[] headers;
         }
     }
+
+    scope event
+    {
+        enum HttpEventType (uint8)
+        {
+            CORS_PREFLIGHT_REQUEST_NOT_ALLOWED (1),
+            CORS_REQUEST_NOT_ALLOWED (2),
```

Comment:
Suggest we follow a similar pattern to authorization but for access control failure only, since there is no requirement to log successful access control (normal case).
```suggestion
            ACCESS_CONTROL_FAILURE (1),
```

Reply:
OK